### DISCUSSION TITLE
Please tell me your favorite tourist destinations in Japan! 好きな日本の観光地を教えてください！

### DISCUSSION THREAD

**Discussion by t-yujiro** [NAIST-SE/AttractingContributors#423] [General]
Please tell me your favorite tourist destinations in Japan! 

あなたの好きな日本の観光地を教えてください！

My fork :
https://github.com/t-yujiro/AttractingContributors-Tourism

**Comment by aru1702** [comment]
I already go to Osaka and Kyoto, now I want to go to Nagoya, Kobe, and parts of Kyushu and Tohoku.
How about you?

See my repo: https://github.com/aru1702/AttractingContributors-TouhouFumo
Hop into my discussion: https://github.com/NAIST-SE/AttractingContributors/discussions/407
